src/app/components/appointments/appointments.component.css

```diff
@@ -0,0 +1,40 @@
+p {
```

Comment:
Is there any reason we are not using common css style file to put all our styles in one place